manifests/agent/config.pp

```diff
@@ -15,24 +11,28 @@
     value   => $puppet::agent::server,
   }
 
+  if $puppet::agent::ca_server {
```

Comment:
We have validate_bool which errors out if we pass "true" instead of true already.